bin/all_sky_search/pycbc_add_statmap

```diff
@@ -247,7 +249,7 @@ fg_times = (f['foreground/%s/time' % ifo][:] for ifo in all_ifos)
 
 # Cluster by statistic value. Currently only clustering zerolag,
 # i.e. foreground, so set all timeslide_ids to zero
-cidx = pycbc.events.cluster_coincs_multiifo(f['foreground/stat'][:],
+cidx = pycbc.events.cluster_coincs_multiifo(f[f'foreground/{args.cluster_stat}'][:],
```

Comment:
I have a minor preference for `['foreground'][args.cluster_stat]`, but it doesn't matter too much

Reply:
ok ... I wrote it this way to be simpler to review as a code change - fwiw we also have `['foreground/ifo/time']` rather than `['foreground'][ifo]['time']`